hicommonwealth/commonwealth
# CE 2: worker directory

## Link to Issue
Closes: #5032 

## Description of Changes
- Created `packages/commonwealth/server/workers/` directory
- Moved/renamed `/server/CommonwealthConsumer/` to `/server/workers/commonwealthConsumer`

## Test Plan
**Locally**
- Ensure `yarn start-consumer` runs successfully (needs RabbitMQ to run)

**QA**
- Check `consumer` logs on Heroku to ensure it started and is running. You can also restart the consumer dyno to double-check.

## Deployment Plan

1. 

## Other Considerations

-
